### PR TITLE
Fixed Hidden Point Primitive Bug

### PR DESCRIPTION
**Pull request by ProjectBarks**
create an entity with point graphics and change its show property to false it doesn't hide, instead, it positions itself in the center of the screen while still being visible.

this issue only occurs when I'm in a Ubuntu environment

Fixed by changing translucency in addition to point location when showing

This is in response to https://github.com/AnalyticalGraphicsInc/cesium/issues/8043

### CODE REVIEW

**Comment by cesium-concierge** [comment]
Thank you so much for the pull request @ProjectBarks! I noticed this is your first pull request and I wanted to say welcome to the Cesium community!

The [Pull Request Guidelines](https://github.com/AnalyticalGraphicsInc/cesium/blob/master/CONTRIBUTING.md#pull-request-guidelines) is a handy reference for making sure your PR gets accepted quickly, so make sure to skim that.

* :x: Missing `CONTRIBUTORS.md` entry.
    * Please add yourself to the [contributors](https://github.com/ProjectBarks/cesium/blob/hidden-point-primitive-fix/CONTRIBUTORS.md) file!
* :heavy_check_mark: Signed CLA found.
* :grey_question: [CHANGES.md](https://github.com/AnalyticalGraphicsInc/cesium/blob/hidden-point-primitive-fix/CHANGES.md) was not updated.
    * If this change updates the public API in any way, please add a bullet point to `CHANGES.md`.
* :grey_question: Unit tests were not updated.
    * Make sure you've [updated tests](https://github.com/AnalyticalGraphicsInc/cesium/tree/master/Documentation/Contributors/TestingGuide) to reflect your changes, added tests for any new code, and ran the code coverage tool.

Reviewers, don't forget to make sure that:

- [ ] Cesium Viewer works.
- [ ] Works in 2D/CV.
- [ ] Works (or fails gracefully) in IE11.

**Comment by OmarShehata** [comment]
Some thoughts:

* Consider hiding the point the same way that point clouds do here: https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Scene/PointCloud.js#L1177-L1178
  * The advantage there is that moving the point off screen produces no fragment, as opposed to rendering it with alpha 0.
* This incorrect line is still there: https://github.com/AnalyticalGraphicsInc/cesium/blob/453b3ff53517e46443ba95748d1c423e6ac19fbb/Source/Shaders/PointPrimitiveCollectionVS.glsl#L94

**Comment by ProjectBarks** [comment]
Thanks omar! Good suggestions.

**Comment by emackey** [comment]
Billboards are similar to points, do they have the same issue?

**Comment by ProjectBarks** [comment]
@emackey it doesn't appear to be an issue. Tested using this [sandbox](http://localhost:8080/Apps/Sandcastle/index.html#c=ZZBNb8IwDIb/StRLi8RcJobKoKBJ7Ii0w6SdKqG0NSWaG1dJWmAT/30pHdI+fLJf+3kdp5NGdAqPaMRKaDyKDVrV1vB21aKwuJYb1k4qjSYcLTOdadF57OSJAQXUTjmFFmRZRp++76Nh6zXWYnHz3EjjfCb1FPaG62esDKKN7pIZzB6TJBmLhwlMpvP5dDQePHJFlLM0pVh8u/ahalmhtw0B4mtu42HDbssV77hDQ/IMja7CAbpk+tI/vGBtmRCIq+jUCyewBz76O/aSLC6DcZBadyZcD9yTqhs2TrSGIr/LYd2Q9CfEeVu8o4PC2t4ljW9QWqpOqHKVBX/+LQtEQdJa39m3RK/qA7NgncZ+/hdGLEulq5fhhH7kcL/eDiIApLEv/1OOmXJpfjh+AQ).

File: CONTRIBUTORS.md [hunk 1]
```diff
@@ -222,3 +222,4 @@ See [CONTRIBUTING.md](CONTRIBUTING.md) for details on how to contribute to Cesiu
 * [Dennis Adams](https://github.com/dennisadams)
 * [Hai Zhou](https://github.com/verybigzhouhai)
 * [Pascal Poulain](https://github.com/ppoulainpro)
+* [Brandon Barker](https://github.com/ProjectBarks)
```
Comment (mramato) [inline CONTRIBUTORS.md]: Same comment as the other PRs, move this to Cesium GS, above.

**Comment by mramato** [comment]
I can confirm this is fixed on my machine.  Thanks @ProjectBarks this one really annoyed me.

@OmarShehata I'll let you handle the rest of this PR, merge when you are ready (or bump to Sean if you need a second opinion).

**Comment by mramato** [comment]
There are a bunch of local test failures in this branch. @ProjectBarks you probably aren't aware of this, but travis  CI doesn't run the WebGL tests, so you should always run tests locally as well. (Assuming they are failing for everyone and not just me)

**Comment by mramato** [comment]
I actually can't reproduce the failures any more, I think I might have been missing that last commit?  Either way this is good to go, @OmarShehata feel free to merge if you don't need a second opinion on the shaders.

**Comment by mramato** [comment]
@OmarShehata did you need to take another look at this or can we merge?

**Comment by OmarShehata** [comment]
Sorry for the delay here. This looks good to me.

**Pull request merged**